src/kernels/variables/debuggerVariables.ts

```diff
@@ -116,7 +119,7 @@ export class DebuggerVariables
         if (this.active) {
             // Note, full variable results isn't necessary for this call. It only really needs the variable value.
             const result = this.lastKnownVariables.find((v) => v.name === name);
-            if (result && kernel?.resourceUri?.fsPath.endsWith('.ipynb')) {
+            if (result && kernel?.resourceUri?.path.endsWith('.ipynb')) {
```

Comment:
This was part of the existing code already and I don't think that it's much of an issue. But since we have the vscode-path path already I think that path.extname is a safer way to get this.

Reply:
I like that! Thank you!